app/views/lessons/_lesson_buttons.html.erb

```diff
@@ -10,6 +10,28 @@
   <%= link_to 'Next Lesson',
       course_lesson_path(course, lesson.next_lesson, ref: 'lnav'),
       title: "Move on to '#{lesson.next_lesson.title}'",
-      class: "button #{next_lesson_button_state(lesson)} lesson-button-group__item lesson-button"
+      class: "button #{next_lesson_button_state(lesson)} lesson-button-group__item lesson-button",
+      id: "next-lesson-button"
   %>
 <% end %>
+ 
+<script>
+  // Fire an event whenever a user marks a lesson completed on the individual lesson page
+  // else marks a lesson uncompleted on the individual lesson page
+  $('#submission-button').on('click', function(e) {
+    if ($(this).data('method') === 'post')  {
+      console.log('completed')
+      _gaq.push(['_trackEvent', 'lesson_completion', 'mark_completed', window.location.href, 1]);
+      
+    } else {
+      console.log('not completed')
+      _gaq.push(['_trackEvent', 'lesson_completion', 'mark_not_completed', window.location.href, 1]);
+    }
+  });
+  //*******
+  // Fire an event when a user clicks on the next lesson navigation link in the individual lesson page
+  $('#next-lesson-button').on('click', function(e) {
```

Comment:
can we use an attribute selector instead of an id here too please?

this should do it
```
[title^="Move on to"]
```